### PR TITLE
Label permission issues

### PR DESCRIPTION
**Pull request by ainhoaL**
We are creating a new Project Area for the Permission Framework and we want to add a new 'permission' label for any issues that relate to the permission framework

### CODE REVIEW

**Review by Rugvip — COMMENTED** [review]
Except for the detail of which exact label we're gonna use this looks good 😁 👍

File: issue-sync/syncIssueLabels.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export async function syncIssueLabels(
     search: 'search',
     catalog: 'catalog',
     scaffolder: 'scaffolder',
+    permission: 'permission',
```
[the remark this answers is not in the document]
Comment (ainhoaL) [reply issue-sync/syncIssueLabels.ts:28]: Ah so not this then: https://github.com/backstage/backstage/compare/master...permission-ownership?quick_pull=1#diff-a22c263686553013feaeb0677d26eeb0b8778a756c4311c1fce13384258026aaR27 😅 that was our first guess when we started adding ownership changes

[the remark this answers is not in the document]
Comment (vinzscam) [reply issue-sync/syncIssueLabels.ts]: renamed to `area:permissions`

[the remark this answers is not in the document]
Comment (ainhoaL) [reply issue-sync/syncIssueLabels.ts]: renamed to `area:permission`

**Review by Rugvip — APPROVED** [review]
Thank you! 👍 

Threw in some more changes while we're at it 😅

**Pull request merged**